web-client/src/app/state/qAccess/q-access.service.ts

```diff
@@ -124,7 +131,10 @@ export class QAccessService implements OnDestroy {
         });
       }
     } catch (error) {
-      console.log(error);
+      await this.notification.swal.fire({
+        icon: 'error',
+        text: 'An unexpected error occured when saving your wallet address.',
```

Comment:
does this error prevent the user from accessing their account? if yes, maybe add a footer saying: please try again